### PR TITLE
"Update the request examples in the api-docs for the 2022 reporting period"

### PR DESCRIPTION
**Pull request by naincygoyal**
I have made these changes https://github.com/GRESB/api-docs/issues/88#issuecomment-1049779332 in https://github.com/GRESB/api-docs/blob/master/source/includes/_assets.md

### CODE REVIEW

File: source/includes/_assets.md [hunk 1]
```diff
@@ -39,59 +39,60 @@ a batch.
...
-          }
-        ],
-        "_outliers": [],
-        "created_at": "2019-01-15T11:07:13.436Z",
-        "updated_at": "2020-01-24T12:05:11.456Z"
-    }
-]
+{
+    "gresb_asset_id": 833335,
+    "country": "CD",
```
Comment (joeyhorst) [inline source/includes/_assets.md:51]: please change it to `NL` to be consistent with a "real"(ish) location

Using the GRESB Office as location and name could be nice.

File: source/includes/_assets.md [hunk 2]
```diff
@@ -39,59 +39,60 @@ a batch.
...
+    "state_province": "flevoland",
+    "city": "franekeradeel",
+    "address": "ABC",
```
Comment (joeyhorst) [inline source/includes/_assets.md:54]: rather use a real address or none at all - this gives a bad example

File: source/includes/_assets.md [hunk 3]
```diff
@@ -39,59 +39,60 @@ a batch.
...
+    "lat": null,
+    "lng": null,
+    "partners_id": null,
+    "construction_year": null,
+    "asset_ownership": null,
+    "certifications": [],
+    "asset_size": "21.0",
+    "annual_data": [
```
Comment (joeyhorst) [inline source/includes/_assets.md:62]: either create a minimum viable asset or fill in all the fields (including `asset_ownership`, `construction_year` etc.

The current example has a lot of yearly data, but is missing some of the asset characteristics. I would suggest to lower the number of annual_data fields.

File: source/includes/_assets.md [hunk 4]
```diff
@@ -100,56 +101,59 @@ Returns the assets of the entity specified in the URL, along with any annual dat
...
 > Response
 
 ```json
 {
-    "gresb_asset_id": 442,
-    "country": "NL",
-    "state_province": "Noord-Holland",
-    "city": "Amsterdam",
-    "address": null,
+    "gresb_asset_id": 833335,
```
Comment (joeyhorst) [inline source/includes/_assets.md:112]: This response will change according to the changes made above.

File: source/includes/_assets.md [hunk 5]
```diff
@@ -164,60 +168,43 @@ curl -X POST https://api.gresb.com/api/v1/entities/5028/assets \
...
-            "asset_size": 500,
-            "property_type_code": "OCHI",
-            "en_tot_lc_te": 91.7,
-            "wat_abs_lc_t": 69.11
+            "year": 2021,
+            "asset_size": 21,
+            "asset_gav":1,
+            "asset_ownership":0,
+            "property_type_code": "OTSS",
+            "asset_name": "APITest2",
```
Comment (joeyhorst) [inline source/includes/_assets.md:187]: use `APITest` for all instead of `APITest2`, `APITest3`, seems more professional that way.

**Review by joeyhorst — CHANGES_REQUESTED** [review]
Main comments are around location and the example used. Probably better to make a smaller example.

**Comment by joeyhorst** [comment]
@naincygoyal what is important and needs to be done in the short run is informing the user that they need to create an account in portal.gresb.com and use api.gresb.com for their base URL when making a request.

Soon we will remove the UI from api.grebs.com.

Find all instances like 

https://github.com/GRESB/api-docs/blob/master/source/includes/_oauth2.md?plain=1#L12-L14

**Comment by naincygoyal** [comment]
@joeyhorst   I have incorporated the review comments , please have a look into it once and let me know if any modification require.
https://github.com/GRESB/api-docs/commit/58a2577f1915f1870d08bf29c1a273994b37d276

**Comment by joeyhorst** [comment]
@naincygoyal you will have to resolve my comments and then push your new commits to this PR in order for me to do a second review.

File: source/includes/_assets.md [hunk 6]
```diff
@@ -13,6 +13,7 @@ both years.
   - construction year
   - building certifications
   - asset ownership
+  - asset size
```
Comment (joeyhorst) [inline source/includes/_assets.md:16]: `asset_size` is part of the annual data, not the building characteristics, although the UI shows it as such.

```
    "annual_data": [
        {
            "year": 2020,
            "asset_name": "The White House",
            "whole_building": true,
            "asset_size": 500
```

File: source/includes/_assets.md (hunk 3 above)
Comment (joeyhorst) [reply source/includes/_assets.md:63]: did you make the example(s) shorter?

**Review by joeyhorst — CHANGES_REQUESTED** [review]
one breaking comment, the other a suggestion.

**Comment by naincygoyal** [comment]
@joeyhorst  These are the new changes which I made please review it once and let me know if any further changes are required.

https://github.com/GRESB/api-docs/pull/89/commits/bce535509b90a0b2b6f52bdb68ee943b6cd5f25e

**Comment by joeyhorst** [comment]
@naincygoyal https://github.com/GRESB/api-docs/commit/bce535509b90a0b2b6f52bdb68ee943b6cd5f25e#r67953623

**Review by joeyhorst — APPROVED** [review]

**Pull request merged**